polynote-kernel/src/main/scala/polynote/kernel/lang/python/PythonInterpreter.scala

```diff
@@ -80,6 +80,22 @@ class PythonInterpreter(val kernelContext: KernelContext) extends LanguageInterp
     } else {
       withJep(task).unsafeRunSync()
     }
+
```

Comment:
These methods are for when you want a list- or dict-like thing to become a Scala `List`/`Map`, but it won't happen automatically because the interpreter only special cases things that are precisely `list` or `dict`. It just coerces to the desired type in python so it will be output in the desired way.